packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -17,4 +41,112 @@ test.describe('PieCookieBanner - Component tests', () => {
         // Assert
         expect(cookieBanner).toBeVisible();
     });
+
+    test('clicking "Accept all" emits the correct event and closes the cookie banner', async ({ mount, page }) => {
+        // Arrange
+        const events : Array<Event> = [];
+
+        await mount(PieCookieBanner, {
+            props: {} as CookieBannerProps,
+
+            on: {
+                [ON_COOKIE_BANNER_ACCEPT_ALL]: (event: Event) => events.push(event),
+            },
+        });
+
+        const cookieBanner = await page.locator(componentSelector);
+
+        // Act
+        await page.locator(acceptAllSelector).click();
+
+        // Assert
+        expect(events).toHaveLength(1);
```

Comment:
Does the name not get picked up? I think i had a similar issue somewhere before but wondered if it was the same thing here too now?